platform_versioned_docs/version-24.1/getting-started/proteinfold.mdx

```diff
@@ -0,0 +1,481 @@
+
+<details>
+  <summary>**nf-core/proteinfold example samplesheet**</summary>
+    
+  | sequence | fasta |
+  | -------- | ----- |
+  | T1024 | https://raw.githubusercontent.com/nf-core/test-datasets/proteinfold/testdata/sequences/T1024.fasta |
+  | T1026 | https://raw.githubusercontent.com/nf-core/test-datasets/proteinfold/testdata/sequences/T1026.fasta |
+
+</details>
+
+In Platform, samplesheets and other data can be made easily accessible in one of two ways: 
+- Use **Data Explorer** to browse and interact with remote data from AWS S3, Azure Blob Storage, and Google Cloud Storage repositories, directly in your organization workspace.
+- Use **Datasets** to upload structured data to your workspace in CSV (Comma-Separated Values) or TSV (Tab-Separated Values) format.
+
+<details>
+  <summary>**Add a cloud bucket via Data Explorer**</summary> 
+
+  Private cloud storage buckets accessible with the credentials in your workspace are added to Data Explorer automatically by default. However, you can also add custom directory paths within buckets to your workspace to simplify direct access.
+
+  To add individual buckets (or directory paths within buckets):
+
+  ![Add public bucket](./_images/data-explorer-add-proteinfold.gif)
+
+  1. From the **Data Explorer** tab, select **Add cloud bucket**. 
+  1. Specify the bucket details:
+      - The cloud **Provider**.
+      - An existing cloud **Bucket path**.
+      - A unique **Name** for the bucket.
+      - The **Credentials** used to access the bucket. For public cloud buckets, select **Public**.
+      - An optional bucket **Description**.
+  1. Select **Add**.
+
+  You can now select data directly from this bucket as input when launching your pipeline, without the need to interact with cloud consoles or CLI tools. 
+
+</details>
+
+<details>
+  <summary>**Add a dataset**</summary>
+
+  From the **Datasets** tab, select **Add Dataset**.
+
+  ![Add a dataset](./_images/proteinfold-dataset.gif)
+
+  Specify the following dataset details:
+
+  - A **Name** for the dataset, such as `proteinfold_samplesheet`.
+  - A **Description** for the dataset.
+  - Select the **First row as header** option to prevent Platform from parsing the header row of the samplesheet as sample data.
+  - Select **Upload file** and browse to your CSV or TSV samplesheet file in local storage, or simply drag and drop it into the box.
+
+  The dataset is now listed in your organization workspace datasets and can be selected as input when launching your pipeline. 
+
+  :::info
+  Platform does not store the data used for analysis in pipelines. The dataset must specify the locations of data stored on your own infrastructure.
+  :::
+
+</details>
+
+## Launch pipeline
+
+:::note
+This guide is based on version 1.1.1 of the nf-core/proteinfold pipeline. Launch form parameters and tools may differ in other versions. 
```

Comment:
```suggestion
This guide is based on [version 1.1.1](https://nf-co.re/proteinfold/1.1.0) of the nf-core/proteinfold pipeline. Launch form parameters and tools may differ in other versions. 
```
Optional, it might be nice to link the version for easy access